### PR TITLE
feat(onboarding): Initial set up for ingestion wizard welcome screen

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

Initial set up for onboarding ingestion wizard screen updates. Nothing is functional yet, just creating the feature flags and this page as an example for now to keep PRs scoped.

Screen Shot 2022-04-18 at 4 16 36 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by clarkus — COMMENTED** [review]
Looks good to me. I do think there is an opportunity to reuse styles from the other onboarding workflows. Paolo did one a while ago at https://github.com/PostHog/posthog/pull/6136. Just pointing it out as an opportunity, not so much a blocking change request.

**Comment by liyiy** [comment]
> Looks good to me. I do think there is an opportunity to reuse styles from the other onboarding workflows. Paolo did one a while ago at #6136. Just pointing it out as an opportunity, not so much a blocking change request.

Yep will keep this in mind 👍

File: frontend/src/scenes/ingestion/constants.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const VERIFICATION = 'VERIFICATION'
 export const WEB = 'Web'
 export const MOBILE = 'Mobile'
 export const BACKEND = 'Backend'
+export const IMPORT = 'Import data via an app'
```
Comment (rcmarron) [inline frontend/src/scenes/ingestion/constants.ts:26]: Nit: I think `APP` might be a more consistent name to match the other options.

**Review by rcmarron — COMMENTED** [review]
The code here all looks good - just a few nits!

I think the screen might be a bit confusing though. It seems like we're asking a few questions at once:
* What type of platform is your app?
* Migrating from another analytics service?
* Just exploring?

As a user, I could be thinking "yes" to all of these, and in which case I don't know what to click. I'd be in favor of breaking these questions up if possible. 

Also, I think "Import data via an app" is odd phrasing. I don't imagine users understand how we use the word "app" at this point. Maybe something like "Import data from a 3rd party" or something like that might be clearer.

**Comment by clarkus** [comment]
> * What type of platform is your app?
> * Migrating from another analytics service?
> * Just exploring?

The copy is a placeholder still - it'll likely change as we get firmer on requirements and what exactly we're calling plugins. Really we're asking if they want to instrument a product directly, import from an existing analytics product, or just demo the product via a bookmarklet. I feel like we should ditch all the demo requirements as they feel completely opposite to our ingestion goals. Mind if we iterate on copy in [figma](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf?node-id=3865:26823#181304197)?

**Review by rcmarron — APPROVED** [review]
> The copy is a placeholder still - it'll likely change as we get firmer on requirements and what exactly we're calling plugins. Really we're asking if they want to instrument a product directly, import from an existing analytics product, or just demo the product via a bookmarklet. I feel like we should ditch all the demo requirements as they feel completely opposite to our ingestion goals. Mind if we iterate on copy in [figma](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf?node-id=3865:26823#181304197)?

Works for me. In that case, just the few nits above, but assuming we work the copy a bit in later iterations, it looks good to me.

File: frontend/src/scenes/ingestion/panels/PlatformPanel.tsx [hunk 2]
```diff
@@ -1,35 +1,79 @@
...
-                        onClick={() => setPlatform(platform as PlatformType)}
-                    >
-                        {platform}
-                    </Button>
-                ))}
-            </Row>
-        </CardContainer>
+        <>
+            {onboarding1 ? (
+                <div style={{ width: '30vw' }}>
```
[the remark this answers is not in the document]
Comment (liyiy) [reply frontend/src/scenes/ingestion/panels/PlatformPanel.tsx:20]: yesss

File: frontend/src/scenes/ingestion/constants.ts (hunk 1 above)
Comment (liyiy) [reply frontend/src/scenes/ingestion/constants.ts:26]: yesss

File: frontend/src/scenes/ingestion/panels/PlatformPanel.tsx [hunk 3]
```diff
@@ -1,35 +1,79 @@
...
+                            Let's get you up and running with PostHog! What type of platform is your app? Migrating from
+                            another analytics service? Try one of our plugins or integrations to ingest data.
+                        </p>
+                        <Col>
+                            {platforms.map((platform) => (
+                                <LemonButton key={platform} fullWidth center type="primary" className="platform-btn">
+                                    {platform}
+                                </LemonButton>
+                            ))}
+                            <LemonButton fullWidth center type="primary" className="platform-btn">
```
[the remark this answers is not in the document]
Comment (liyiy) [reply frontend/src/scenes/ingestion/panels/PlatformPanel.tsx:36]: ehhh I should probably move the `platforms` array into `ingestionLogic` then if we're dynamically updating its value, I'll do that once I see how the rest of the ingestion constants work out

**Pull request merged**